BTCPayServer.Client/Permissions.cs

```diff
@@ -28,6 +28,7 @@ public class Policies
         public const string CanViewNotificationsForUser = "btcpay.user.canviewnotificationsforuser";
         public const string CanViewUsers = "btcpay.server.canviewusers";
         public const string CanCreateUser = "btcpay.server.cancreateuser";
+        public const string CanManageUser = "btcpay.server.canmanageuser";
```

Comment:
```suggestion
        public const string CanManageUsers = "btcpay.server.canmanageusers";
```

Better plural so that it is clearer that it is a server policy, else it sounds like a current user edit permission

Reply:
I was wondering about this. Because just above it's singular.

Reply:
Made it plural, even if it doesn't match above, it's more logical.